samples/net/ptp/sample.yaml

```diff
@@ -0,0 +1,21 @@
+common:
+  harness: net
+  tags:
+    - net
+    - ptp
+sample:
+  description: Test PTP functionality
+  name: PTP sample app
+tests:
+  sample.net.ptp:
+    platform_allow:
+      - native_posix
+      - native_posix/native/64
+      - native_sim
+      - native_sim/native/64
```

Comment:
If you allow the 64 bit versions, I guess you also need a configuration overlay for them, or?